clients/include/testing_trsv_batched.hpp

```diff
@@ -157,51 +158,91 @@ hipblasStatus_t testing_trsv_batched(const Arguments& argus)
     hipMemcpy(dx_or_b, bx_or_b, sizeof(T*) * batch_count, hipMemcpyHostToDevice);
 
     /* =====================================================================
-           ROCBLAS
+           HIPBLAS
     =================================================================== */
-    if(argus.timing)
-    {
-        gpu_time_used = get_time_us(); // in microseconds
-    }
-
-    for(int iter = 0; iter < 1; iter++)
+    if(argus.unit_check || argus.norm_check)
     {
+        status = hipblasSetPointerMode(handle, HIPBLAS_POINTER_MODE_HOST);
+        if(status != HIPBLAS_STATUS_SUCCESS)
+        {
+            hipblasDestroy(handle);
+            return status;
+        }
```

Comment:
Might be worth converting this to a define, status = HIPBLAS_CHECK( hipBlasFunctionxyz ) or other.